From a80753ac9a7300f5293acb978f26b8479cfa16c7 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Sun, 10 Mar 2019 11:52:43 +0200
Subject: [PATCH 026/386] ARM64: dts: Increase the PCIe address space for
 Macchiato BIN

Adjust the PCIe address space of MacchiatoBin board to match
the TF-A address mapping.
This patch will allow usage of PCIe cards with higher IO space
requiremens (like video adapters and/or high speed network cards)

Change-Id: I8443f234e542ca6f5bf0b4f7ea7e44dadaa90e93
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/5442
Tested-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 70dad1c7d373..136abe341378 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -180,6 +180,10 @@
 	num-viewport = <8>;
 	reset-gpio = <&cp0_gpio1 20 GPIO_ACTIVE_LOW>;
 
+	ranges = <0x1000000 0x0 0x00000000 0x0 0xeff00000 0x0 0x00010000>,
+		 <0x2000000 0x0 0xc0000000 0x0 0xc0000000 0x0 0x20000000>,
+		 <0x3000000 0x8 0x00000000 0x8 0x00000000 0x1 0x00000000>;
+
 	/* Generic PHY, providing serdes lanes */
 	phys = <&cp0_comphy0 0
 		&cp0_comphy1 0
-- 
2.17.1

